test/error/error.test.js

```diff
@@ -1,145 +1,83 @@
 'use strict'
 
-var DOMParser = require('../../lib/dom-parser').DOMParser
-const assert = require('../assert')
+const { DOMParser } = require('../../lib/dom-parser')
 
-var xml =
-	'<scxml xmlns="http://www.w3.org/2005/07/scxml" version="1.0"\n\
```

Comment:
I wonder if this is something that we should "restore" testing?

Reply:
Done